<!--This launch file controls the plane to be level. -->
<launch>
 <arg name="mav_name"  default="MTD"/>
 <rosparam command="load" file="$(find rosplane)/param/MTD.yaml" ns="controller"/>
 <node name="naze"  pkg="rosflight"  type="rosflight_io" output="screen">
   <param name="port" value="/dev/naze"/>
 </node>
 <node name="uINS" pkg="inertial_sense" type="inertial_sense_node" output="screen">
   <param name="~port" value="/dev/uINS"/>
   <param name="~lat_ref" value="39.969582"/> <!-- 40.2679779 -->
   <param name="~lon_ref" value="-111.96711"/> <!-- -111.6344569 -->
   <param name="~h_ref" value="1426.681"/> <!--1470.0 -->
 </node>

 <node name="estimator" pkg="rosplane" type="rosplane_estimator" output="screen">
   <param name="~use_inertial_sense" type="bool" value="true"/>
 </node>

 <node name="controller" pkg="rosplane" type="rosplane_controller" output="screen">
   <param name="PWM_RAD_E" value="2.8"/>
   <param name="PWM_RAD_A" value="-1.6"/>
   <param name="PWM_RAD_R" value="1.8"/>
   <param name="ALT_TOZ" value="20"/>
   <param name="ALT_HZ" value="10"/>
   <param name="MAX_T" value="1.0"/>
   <param name="groundD" value="0.0"/>
 </node>
</launch>
